src/main.rs

```diff
@@ -303,7 +303,8 @@ fn reverse(
                     .get(language)
                     .and_then(|lang| lang.block_labels.as_ref())
                 {
-                    let source = fs::read_to_string(&file).map_err(|err| err.to_string())?;
+                    let source = fs::read_to_string(&file)
+                        .map_err(|err| format!("{}: {}", err.to_string(), file.display()))?;
```

Comment:
The call to `.to_string()` is unnecessary (but creates a temporary `String`) as you formatting this `err` into a `String` anyway via `format!`, e.g.
```rust
format!("{}: {}", err, file.display())
```